app/client/cli/keys.go

```diff
@@ -405,6 +374,10 @@ func keysImportCommands() []*cobra.Command {
 						return err
 					}
 				case "raw":
+					// it is unarmoured so we need to confirm the passphrase
```

Comment:
I believe we should have the user confirm the passphrase anytime we are armoring a private key interactively.

Reply:
But they already have to provide either the password as a flag or type it in any way? Maybe this is useful for creation but elsewhere its already being confirmed - this is just my opinion though. Whatever you think fits best for the use case

Reply:
idk, i can't think of one CLI tool that doesn't ask you to confirm a password input interactively. seems like a deviation from standard password input ux.

Reply:
My only thought is that unless the `--pwd` flag is being used the `pwd  = readPassphrase(pwd)` will cause a password prompt by the user anyway. Having to do this back to back seems excessive outside of key creation. But this is really not a huge deal, if you think this matches the general UX from other CLI apps then I'll go with your way 🔥

Reply:
Exactly, this `readPassphrase` should always be followed by a confirm if it's a new passphrase (else we should just force one passphrase for the whole keybase). 

decrypting never requires a confirmation since the feedback is immediate if it's wrong.

Confirmation prompts should be required any time things are being encrypted interactively. Just my thoughts on this matter though, I guess i'm basing this totally on my own experience.

Reply:
Yup, that is what's happening here. Confirm passphrase for the newly imported raw private key string if running interactively.

Reply:
Per my review, let's update the comment.

Reply:
sry i missed this one, wdym?